Documentation/pretty-formats.txt

```diff
@@ -206,6 +208,8 @@ The placeholders are:
 '%ci':: committer date, ISO 8601-like format
 '%cI':: committer date, strict ISO 8601 format
 '%cs':: committer date, short format (`YYYY-MM-DD`)
+'%ch':: committer date, human style(like the `--date=human` option of
```

Comment:
I doubt @gitgitgadget will like this, but you're missing a ` `.
```suggestion
'%ch':: committer date, human style (like the `--date=human` option of
```

Reply:
Thanks :), I think this small typo error can directly tell Junio, 
he will deal with it kindly in next branch.

Reply:
@gitster